README.md

```diff
@@ -33,3 +33,40 @@ new EditorView({
 });
 
 ```
+
+### Options
+
+You can provide an options object to `indentationMarkers()` with the following
+optional properties:
+
+- `activeBlockMarkers`
+    
+    Boolean that controls whether the active block markers are styled
+    differently. Setting this to `false` provides a significant performance
+    enhancement because it means that markers do not need to be regenerated
+    when the selection changes. Defaults to `true`.
+ 
+- `columnZeroMarkers`
```

Comment:
in a similar vain to the above, wdyt about negating this and calling it something like `hideFirstIndent`? think that makes it more clear what's going on as "zero markers" is a little more ambiguous.

Reply:
Definitely agree about "zero markers" being ambiguous. I'd probably prefer "show" to "hide" but don't feel strongly about it. I've gone with `showFirstIndent` but will very happily change it to `hideFirstIndent` if you'd prefer it.

Reply:
great, thanks! yeah I think I'd prefer `hideFirstIndent` since you have to take an explicit action to do so, so that feels more like what's being configured as they are shown by default.